torch/csrc/distributed/rpc/tensorpipe_utils.cpp

```diff
@@ -311,8 +311,9 @@ c10::intrusive_ptr<Message> tensorpipeDeserialize(
     tensors.emplace_back(std::move(t));
   }
 
-  for (const auto i : c10::irange(tpDescriptor.tensors.size())) {
-    auto& tensor = tpDescriptor.tensors[i];
+  size_t tpDescriptorPos = 0;
```

Comment:
Pos->Index 

To stay consistent with the rest of the file, e.g., 

https://github.com/pytorch/pytorch/blob/22bae0d12c2c1f645bb8d0bb95ad3b33b500e343/torch/csrc/distributed/rpc/tensorpipe_utils.cpp#L296

Reply:
Okay.